ixmp/backend/base.py

```diff
@@ -742,14 +790,19 @@ def item_get_elements(self, s: Scenario, type, name, filters=None):
             When *type* is 'set' and *name* an index set (not indexed by other
             sets).
         dict
-            When *type* is 'equ', 'par', or 'set' and *name* is scalar (zero-
+            When *type* is 'equ', 'par', or 'var' and *name* is scalar (zero-
```

Comment:
not sure, but maybe `'equ', 'par', 'var', or 'set'`?

Reply:
There are no scalar (0-dimensional) sets in the data model. An index set is always 1-D; an indexed set is indexed by at least 1 index set, so it is ≥1-D.

Reply:
Sorry, I didn't see at first it is changing description of one of return types